zeppelin-interpreter/src/main/thrift/RemoteInterpreterService.thrift

```diff
@@ -56,6 +56,11 @@ struct RemoteInterpreterEvent {
   2: string data      // json serialized data
 }
 
+struct InterpreterCompletion {
+  1: string name,
+  2: string value
```

Comment:
@bzz Regarding the two value `name` and `value`, can [this comment](https://github.com/apache/zeppelin/pull/984#issuecomment-225249183) be an answer for your question ?